model/eligibility/Eligibility.php

```diff
@@ -138,15 +139,25 @@ public function getId()
     }
 
     /**
-     * @return array
+     * @return bool
      * @throws \core_kernel_persistence_Exception
      */
+    public function isProctored()
+    {
+        $isProctorBypassed =  $this->getResource($this->id)->getOnePropertyValue(new Property(EligibilityService::PROPERTY_BYPASSPROCTOR_URI));
```

Comment:
It's better to use `$this->getProperty(EligibilityService::PROPERTY_BYPASSPROCTOR_URI)` here instead of using new Property(). In this case you can mock it in unit tests